.kitchen.digitalocean.yml

```diff
@@ -0,0 +1,7 @@
+driver:
+  name: digitalocean
+  private_networking: false
+  size: 2gb
+
+transport:
+  name: rsync
```

Comment:
Speedier sync when working with a slow connection and getting the tests working.